### ISSUE TITLE
Cerebral lesion

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6858]
**Preferred term label:**
Cerebral lesion

**Synonyms**

**Definition (free text, please give PubMed ID)**
Lesions localised in the cerebrum.
PMID: 8916157

**Parent term (use hpo.jax.org/app)**
Abnormal cerebral morphology

**Comment by pnrobinson** [comment]
@johannaest This is so general that it seems better just to use "Abnormal cerebral morphology", because "lesion" can mean just about anything. Could you comment is you see a distinction and if so what that is?

**Comment by johannaest** [comment]
I agree, it could be used as a synonym.

**Comment by pnrobinson** [comment]
added
